app/src/main/kotlin/com/mapbox/vision/teaser/view/ViewUtils.kt

```diff
@@ -1,15 +1,26 @@
 package com.mapbox.vision.teaser.view
 
 import android.view.View
+import android.view.View.GONE
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
 
 fun View.hide() {
-    visibility = View.GONE
+    visibility = GONE
 }
 
 fun View.show() {
-    visibility = View.VISIBLE
+    visibility = VISIBLE
 }
 
 fun View.invisible() {
-    visibility = View.INVISIBLE
+    visibility = INVISIBLE
+}
+
+fun View.toggleVisibleGone() {
+    if (visibility == GONE) show() else hide()
+}
+
+fun View.toggleVisibleInvisible() {
+    if (visibility == INVISIBLE) show() else invisible()
```

Comment:
It'll also switch GONE to INVISIBlE - guess not strictly correct

Reply:
fixed